recipes-framework/tensorflow/tensorflow_2.5.2.bb

```diff
@@ -45,3 +45,7 @@ PROVIDES += " \
     ${@ d.getVar('PROVIDES_FULL') if not d.getVar('TENSORFLOW_LITEONLY') == '1' else ''}"
 
 DEB_BUILD_PROFILES += "${@ 'nofull' if d.getVar('TENSORFLOW_LITEONLY') == '1' else ''}"
+
+python do_unpack_append() {
+    os.remove(d.getVar('S') + "/.bazelversion")
```

Comment:
We should better not put that file into the `.orig.tar.gz`. Then we can simply remove the call in `override_dh_auto_build`.

Reply:
We don't put it in the orig-tarball, already today. That's because the tarball is built only after `override_dh_auto_build` was run (in case we don't use a cached tarball).

But I'm fine with removing that rules statement along this.

Reply:
When using the dh_sequence (which calls `override_dh_auto_build`), we need an orig tarball anyways. The fetching of sources at build time is prohibited by the fake proxy.

My problem with the approach from above is, that we split the fetching post-processing across `debian/rules local` and bitbake.

Reply:
We duplicate it with my current patch.